packages/react-aria-components/src/useDragAndDrop.tsx

```diff
@@ -105,13 +113,18 @@ export function useDragAndDrop(options: DragAndDropOptions): DragAndDrop {
       getItems,
       renderDragPreview,
       renderDropIndicator,
-      dropTargetDelegate
-     } = options;
+      dropTargetDelegate,
+      isDisabled
+    } = options;
+
+    if (isDisabled) {
+      options.acceptedDragTypes = [];
+    }
```

Comment:
We can get rid of this now that useDroppableCollectionState handles returning 'cancel` in the `defaultDropOperation` for us when `isDisabled = true` right?